src/locales/en.json

```diff
@@ -217,6 +217,11 @@
     "title": "New paper%{name}"
   },
   "Scan": {
+    "modal": {
+      "cozySelect": "Select in my Cozy",
```

Comment:
I'm not sure about this one. Maybe `Select from my cozy`? We should google that.